app/dashboard/views.py

```diff
@@ -3074,6 +3080,23 @@ def authenticate_idena(request, handle):
         }
     })
 
+@login_required
```

Comment:
^ if you use the `login_required` decorator -> you don't need the `is_logged_in_user` checked